site/documentation/content/user-guide/coap-adapter.md

```diff
@@ -0,0 +1,231 @@
++++
+title = "CoAP Adapter"
+weight = 210
++++
+
+The CoAP protocol adapter exposes [CoAP](https://tools.ietf.org/html/rfc7252) based endpoints for Eclipse Hono&trade;'s south bound Telemetry, Event and Command & Control APIs.
+<!--more-->
+
+## Device Authentication
+
+The CoAP adapter by default requires clients (devices or gateway components) to authenticate during connection establishment. The adapter supports currently only [PSK](https://tools.ietf.org/html/rfc4279) as part of a DTLS handshake for that purpose. The other variants mentioned in [Securing CoAP](https://tools.ietf.org/html/rfc7252#section-9) may follow in the future.
+
+The adapter tries to authenticate the device using this mechanisms
+
+### PSK
+
+The *identity* provided in the ClientKeyExchange must have the form *auth-id@tenant*, e.g. `sensor1@DEFAULT_TENANT`. The adapter performs the handshake using the credentials `configured Credentials service - (documentation pending)` has one record for the client. The adapter uses the Credentials API's *get* operation to retrieve the credentials on record with the *tenant* and *auth-id* provided by the device in the *identity* and `psk` as the *type* of secret as query parameters.
```

Comment:
the second sentence seems to be garbled ...

Reply:
Please verify, that the new version is not that garbled